src/main/java/me/rigamortis/seppuku/impl/gui/hud/component/PlayerPreviewComponent.java

```diff
@@ -0,0 +1,68 @@
+package me.rigamortis.seppuku.impl.gui.hud.component;
+
+import me.rigamortis.seppuku.api.gui.hud.component.DraggableHudComponent;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.GuiChat;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.OpenGlHelper;
+import net.minecraft.client.renderer.RenderHelper;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.entity.player.EntityPlayer;
+import org.lwjgl.opengl.GL11;
+
+/**
+ * @author cookiedragon234 10/Dec/2019
+ */
+public class PlayerPreviewComponent extends DraggableHudComponent {
+	
+	private static final int INVENTORY_WIDTH = 49;
+	private static final int INVENTORY_HEIGHT = 70;
```

Comment:
So the idea behind changing the width and height was the center the entire player inside of a rectangle.  By idea of using the inventory as a similar size, you get the idea that you want the player to be inside of the box.